### PR TITLE
various small tweaks to #1642

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request
a number of small fixes and balance changes

## Why It's Good For The Game

fixes and balance and shit

## Changelog
:cl:
add: shields can be repaired with a welding tool
tweak: normal briefcases can now block like secure ones can
tweak: fire axe is now worse at blocking, but better at dismemberment and breaking through shields and blocks
tweak: changes up dismemberment calculations to work far better.
tweak: powerfist is no longer snowflake bullshit that fucks the entirety of all melee combat conventions by bypassing shields and armor completely, but it now actually works as a destroyer of shields and the true best melee weapon in the game (not really)(but close)
tweak: penetrating projectiles now bypass shields
balance: projectiles deal full damage to blocks
balance: nerfed toy katana
balance: nerfed chairs
balance: nolimbdisable doesnt let you block forever
balance: stamina healing drugs make you far worse at blocking
balance: cult dagger no longer blocks as if it's a longsword, cult shield is now less durable
fix: energy shield is now repaired when you turn it back on after it breaks. However, after breaking, there is a cooldown before you can use it again
fix: double esword spin is now less theatrical, making it no longer disrupt your blocks.
fix: active blocking now works as intended
fix: projectile blocking now works as intended
/:cl:

### CODE REVIEW

**Comment by zeskorion** [comment]
Adding do not merge till i fix some more pressing issues with this

**Comment by PowerfulBacon** [comment]
Ping me when it is ready

**Comment by zeskorion** [comment]
> Ping me when it is ready

aight

File: code/game/objects/items/twohanded.dm [hunk 1]
```diff
@@ -372,12 +371,9 @@
...
 
-/obj/item/twohanded/dualsaber/proc/jedi_spin(mob/living/user)
-	for(var/i in list(NORTH,SOUTH,EAST,WEST,EAST,SOUTH,NORTH,SOUTH,EAST,WEST,EAST,SOUTH))
-		user.setDir(i)
-		if(i == WEST)
-			user.emote("flip")
-		sleep(1)
+/obj/item/twohanded/dualsaber/proc/jedi_spin(mob/living/user) //rip complex code, but this fucked up blocking
+	user.emote("flip")
+	sleep(1)
```
Comment (PowerfulBacon) [inline code/game/objects/items/twohanded.dm]: Is this sleep need since there is no loop anymore?

Comment (zeskorion) [reply code/game/objects/items/twohanded.dm]: nah not really

**Comment by PowerfulBacon** [comment]
image

File: code/game/objects/items/shields.dm [hunk 2]
```diff
@@ -36,13 +36,28 @@
...
 			T.visible_message("<span class='warning'>[hitby] destroys [src]!</span>")
+			obj_integrity = 1
 			shatter(owner)
 			return FALSE
 		take_damage(attackforce * ((100-(block_power))/100))
 		return TRUE
 	else
 		return ..()
 
+/obj/item/shield/attackby(obj/item/weldingtool/W, mob/living/user, params)
```
Comment (PowerfulBacon) [inline code/game/objects/items/shields.dm:47]: if(!istype(W))
    return

You don't need a welder to repair it
image

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]

File: code/game/objects/items/shields.dm [hunk 3]
```diff
@@ -219,12 +235,17 @@
...
+	var/cooldown_timer
 
 /obj/item/shield/energy/shatter(mob/living/carbon/human/owner)
-	playsound(owner, 'sound/effects/turbolift/turbolift-close.ogg', 50, 0, -1)
+	playsound(owner, 'sound/effects/turbolift/turbolift-close.ogg', 200, 1)
 	src.attack_self(owner)
 	to_chat(owner, "<span class='warning'>The [src] overheats!.</span>")
-	
+	cooldown_timer = world.time + cooldown_duration
+	sleep(100)
```
Comment (PowerfulBacon) [inline code/game/objects/items/shields.dm]:
```suggestion
	sleep(cooldown_duration)
```

**Review by PowerfulBacon — APPROVED** [review]

**Comment by ike709** [comment]
what could go wrong

**Pull request merged**